.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!-- Detailed description of the PR -->
+#### PR Summary
+
+<!-- PR Checklist -->
+### Checklist
+  - [ ] This pull request describes why these changes were made.
+
+  - [ ] This PR is into the correct branch.
+
+  - [ ] JIRA ticket links to this PR
```

Comment:
I wonder if we care about a link from JIRA to PR. Or it's more important to have a link from PR to JIRA

Reply:
I just copied from bonnie. I can update it however we want.

Reply:
I don't see the following item:
>JIRA ticket for this PR:
which is opposite to 
> JIRA ticket links to this PR
I usually use "JIRA ticket for this PR" instead of "JIRA ticket links to this PR"